public/views/hackmd/body.ejs

```diff
@@ -234,8 +234,8 @@
                 <h4 class="modal-title" id="myModalLabel"><%= __('Are you sure?') %></h4>
             </div>
             <div class="modal-body" style="color:black;">
-                <h5 class="ui-delete-modal-msg">Do you really want to delete this note?</h5>
-                <strong class="ui-delete-modal-item">All users will lost their connection.</strong>
+                <h5 class="ui-delete-modal-msg"><%= __('Do you really want to delete this note?')</h5>
+                <strong class="ui-delete-modal-item"><%= __('All users will lose their connection.')</strong>
```

Comment:
You missed closing the `=%>`.

Please fix

Reply:
wow, no idea how I missed that